errors/errors.go

```diff
@@ -27,15 +27,15 @@ import (
 func GetErrorMessage(ctx context.Context, err error) error {
 	logger := jsonlog.LoggerFromContextOrDefault(ctx)
 	var formattedErr error
-	switch err.(type) {
+	switch err := err.(type) {
```

Comment:
This modification debatable

Reply:
How, exactly ? It seems to me like it simplifies the code nicely (it's probably also a minor optimization now that I think about it) compared to manual type assertions for every case... Though if you want, I can change the variable name if you find that it makes things unclear

Reply:
After a second look, I do agree with your changes. My bad.